-- Add the cmake module path and include YACMA Python setup macros so that
-- the pyaudi subdirectory can be configured standalone (the ports framework
-- invokes CMake on pyaudi/CMakeLists.txt directly rather than the project
-- root).
--- CMakeLists.txt.orig	2026-08-01 03:44:55 UTC
+++ CMakeLists.txt
@@ -1,3 +1,9 @@
+cmake_minimum_required(VERSION 3.18.0)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/../cmake_modules" "${CMAKE_SOURCE_DIR}/../cmake_modules/yacma")
+
+include(YACMAPythonSetup)
+
 # Python version check.
 if(${PYTHON_VERSION_MAJOR} LESS 3)
     message(FATAL_ERROR "Minimum supported Python version is 3.0")
